identity/src/test/java/com/android/identity/credential/NameSpacedDataTest.java

```diff
@@ -0,0 +1,78 @@
+package com.android.identity.credential;
+
+import androidx.annotation.NonNull;
+
+import com.android.identity.util.CborUtil;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class NameSpacedDataTest {
+
+    static void
+    checkNameSpaced(@NonNull NameSpacedData nameSpacedData) {
```

Comment:
would be good to add asserts for all functions, like `hasDataElement` and `getDataElement` (even though it's indirectly tested via the other gets)

Reply:
Good point, added tests for `hasDataElement()`.